make CMake script for CHOLMOD and UMFPACK consider SUITESPARSECONFIG_DIR when looking for libsuitesparseconfig.a
required because of changes made in SuiteSparse 4.3.0
cfr. https://github.com/FEniCS/dolfin/issues/2
author: Kenneth Hoste (HPC-UGent)
--- dolfin-1.6.0/cmake/modules/FindUMFPACK.cmake.orig	2016-01-12 14:49:31.660886973 +0100
+++ dolfin-1.6.0/cmake/modules/FindUMFPACK.cmake	2016-01-12 15:20:07.597039273 +0100
@@ -36,7 +36,7 @@
 
 # Check for SUITESPARSECONFIG library
 find_library(SUITESPARSECONFIG_LIBRARY suitesparseconfig
-  HINTS ${UMFPACK_DIR}/lib $ENV{UMFPACK_DIR}/lib $ENV{PETSC_DIR}/lib
+  HINTS ${SUITESPARSECONFIG_DIR} ${UMFPACK_DIR}/lib $ENV{UMFPACK_DIR}/lib $ENV{PETSC_DIR}/lib
   DOC "The SUITESPARSE library")
 mark_as_advanced(SUITESPARSECONFIG_LIBRARY)
 
--- dolfin-1.6.0/cmake/modules/FindCHOLMOD.cmake.orig	2015-07-28 17:05:55.000000000 +0200
+++ dolfin-1.6.0/cmake/modules/FindCHOLMOD.cmake	2016-01-12 15:19:12.765414768 +0100
@@ -84,7 +84,7 @@
 
 # Check for SUITESPARSECONFIG library
 find_library(SUITESPARSECONFIG_LIBRARY suitesparseconfig
-  HINTS ${CHOLMOD_DIR}/lib ${CCOLAMD_DIR}/lib $ENV{CHOLMOD_DIR}/lib
+  HINTS ${SUITESPARSECONFIG_DIR} ${CHOLMOD_DIR}/lib ${CCOLAMD_DIR}/lib $ENV{CHOLMOD_DIR}/lib
   $ENV{CCOLAMD_DIR}/lib $ENV{PETSC_DIR}/lib
   DOC "The SUITESPARSECONFIG library")
 
